Resources/config/services.yml

```diff
@@ -1,27 +1,15 @@
 services:
-    Geocoder\Dumper\GeoArray:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
-
-    Geocoder\Dumper\GeoJson:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
-
-    Geocoder\Dumper\Gpx:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
-
-    Geocoder\Dumper\Kml:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
-
-    Geocoder\Dumper\Wkb:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
-
-    Geocoder\Dumper\Wkt:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
+    _instanceof:
```

Comment:
Is this needed?

Reply:
Yes, just simplifying configuration instead of repeating